README.md

```diff
@@ -121,6 +121,17 @@ directory as the `pa_matchbatch.sh` script.
 `OVERRIDE_PATH` indicates where to find version overrides. See next section. It defaults to
 `/tmp/version_overrides.txt`.
 
+`IGNORE_REPOS` contains a (space-separated) list of repos that we do not want to process, i.e.
+run ticketmatch on. Each repo should be specified by their foss name (e.g. "puppet-agent" instead
+of "puppet-agent-private") since the script internally figures out whether to clone a foss or private
+fork. For example, IGNORE_REPOS="puppet-agent facter pxp-agent" will not run ticketmatch on puppet-agent,
+facter and pxp-agent. It defaults to empty.
+
+`ONLY_ON` contains a (space-separated) list of foss repos that we exclusively want to process, ignoring 
```

Comment:
This is useful when I want to run pa_matchbatch.sh on one component. In my workflow when I am the release lead, I like to process things one at a time.